x/emissions/keeper/msgserver/msg_server_losses.go

```diff
@@ -77,9 +76,14 @@ func (ms msgServer) InsertBulkReputerPayload(
 	lossBundlesByReputer := make(map[string]*types.ReputerValueBundle)
 	latestReputerScores := make(map[string]types.Score)
 	for _, bundle := range msg.ReputerValueBundles {
+		if err := bundle.Validate(); err != nil {
+			fmt.Println("Error validating reputer value bundle: ", err)
```

Comment:
we should move away from using println we already print way too much 

should probably using `"cosmossdk.io/log"` instead

anyways the correct thing to do would be to flag some kind of error in the `MsgInsertBulkReputerPayload` and return an error from this function

Reply:
Note I am not saying to not continue here, I am saying to continue _and also return error_ this code is right now silently failing

Reply:
same with all the other continue cases below

Reply:
+1  I think the best alternative is logging using the `ctx` 

e.g `ctx.Logger().Info(fmt.Sprintf("Error validating reputer value bundle: %s", err))`